src/testsuite/arith128_test_i64.c

```diff
@@ -556,179 +746,179 @@ test_popcntd (void)
 
   printf ("\ntest_popcntd Vector Pop Count doubleword\n");
 
-  i = (vui64_t )CONST_VINT64_DW(0, 0);
-  e = (vui64_t )CONST_VINT64_DW(0, 0);
+  i = (vui64_t) CONST_VINT64_DW (0, 0);
+  e = (vui64_t) CONST_VINT64_DW (0, 0);
   j = vec_popcntd(i);
 
 #ifdef __DEBUG_PRINT__
   print_vint128 ("popcntd({0, 0) ", (vui128_t)j);
 #endif
   rc += check_vuint128x ("vec_popcntd:", (vui128_t)j, (vui128_t) e);
 
-  i = (vui64_t )CONST_VINT64_DW(0, -1);
-  e = (vui64_t )CONST_VINT64_DW(0, 64);
+  i = (vui64_t) CONST_VINT64_DW (0, -1);
+  e = (vui64_t) CONST_VINT64_DW (0, 64);
   j = vec_popcntd(i);
 
 #ifdef __DEBUG_PRINT__
   print_vint128 ("popcntd({0, -1) ", (vui128_t)j);
 #endif
   rc += check_vuint128x ("vec_popcntd:", (vui128_t)j, (vui128_t) e);
 
-  i = (vui64_t )CONST_VINT64_DW(-1, 0);
-  e = (vui64_t )CONST_VINT64_DW(64, 0);
+  i = (vui64_t) CONST_VINT64_DW (-1, 0);
+  e = (vui64_t) CONST_VINT64_DW (64, 0);
   j = vec_popcntd(i);
 
 #ifdef __DEBUG_PRINT__
   print_vint128 ("popcntd({-1, 0) ", (vui128_t)j);
 #endif
   rc += check_vuint128x ("vec_popcntd:", (vui128_t)j, (vui128_t) e);
 
-  i = (vui64_t )CONST_VINT64_DW(-1, -1);
-  e = (vui64_t )CONST_VINT64_DW(64, 64);
+  i = (vui64_t) CONST_VINT64_DW (-1, -1);
+  e = (vui64_t) CONST_VINT64_DW (64, 64);
   j = vec_popcntd(i);
 
 #ifdef __DEBUG_PRINT__
   print_vint128 ("popcntd({-1, -1) ", (vui128_t)j);
 #endif
   rc += check_vuint128x ("vec_popcntd:", (vui128_t)j, (vui128_t) e);
 
-  i = (vui64_t )CONST_VINT64_DW(1, 8589934596);
-  e = (vui64_t )CONST_VINT64_DW(1, 2);
+  i = (vui64_t) CONST_VINT64_DW (1, 8589934596);
```

Comment:
would this be easier to understand if that big magic number was in hex? _global_, as there are more big magic numbers below

Reply:
maybe but the __DEBUG_PRINT__ below is also decimal. So would need to change a all the places used and there is a good chance of mistake. 

Lets defer this for a later and separate change.